docroot/themes/contrib/civictheme/civictheme-library/components/00-base/_variables.base.scss

```diff
@@ -288,6 +288,7 @@ $civictheme-typography-default: (
     'xxs': ($civictheme-font-base-size * 1.25, $civictheme-font-base-line-height * 1.185, 400, 'primary', -0.15px),
     'm': ($civictheme-font-base-size * 1.5, $civictheme-font-base-size * 2.5, 400, 'primary', -0.35px)
   ),
+  'date': ($civictheme-font-base-size * 0.875, $civictheme-font-base-line-height * 1.25, 600, 'primary', -0.1px),
```

Comment:
could you please remove this addition and find the "closest" text style. Probably should be "label-regular" as this is what we have been using for all non-content text.

we are trying to keep typography sets to a minimum